### ISSUE TITLE
In refactoring Experiment class, should be easier to adjust which targets we're looking at

### ISSUE THREAD

**Issue by avictorious** [fpsim/fpsim#152]
The calibration target keys are quite embedded through the class and are not easy to remove ones you're not interested in.  The flag at the top of the file don't actually function to turn off the target.

**Comment by MObrien-IDM** [comment]
@emilydriano Can you confirm that this is still an issue with v2.0?

**Comment by emilydriano** [comment]
@MObrien-IDM yes, this is still something that we're going to want to update for v2.0. The targets are embedded in the code but we'll want to isolate them for the user as a configurable list, for example, as an argument for the Experiment class
